flip/views.py

```diff
@@ -319,6 +319,29 @@ def get_success_url(self):
         return reverse('study_outcomes_detail', kwargs={'pk': self.study.pk})
 
 
+class UserEntriesView(LoginRequiredMixin,
+                  generic.ListView):
+
+    model = models.Study
+    template_name = 'user_entries.html'
+
+    def get(self, request, *args, **kwargs):
+        request.session['last_viewed'] = time.time()
```

Comment:
Drop, but move it to activities/evaluations listing views (if it isn't already there).